<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomTabForImage</class>
 <widget class="QWidget" name="CustomTabForImage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1299</width>
    <height>609</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGraphicsView" name="originalGrahpic">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>514</width>
     <height>514</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="processedGraphic">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>60</y>
     <width>514</width>
     <height>514</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="originalPhotoLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>514</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Obraz wzorcowy</string>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="transformedPhotoLabel">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>10</y>
     <width>514</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Obraz przetworzony</string>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="transformButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>530</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Przetwórz obraz</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>100</y>
     <width>201</width>
     <height>381</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="toolLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetFixedSize</enum>
    </property>
   </layout>
  </widget>
  <widget class="QComboBox" name="toolsComboBox">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>60</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="currentText">
    <string>Wybierz narzędzie</string>
   </property>
   <item>
    <property name="text">
     <string>Wybierz narzędzie</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Jasność</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Kontrast</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Negatyw</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Filtr - średnia</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Filtr - mediana</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Histogram</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Filtr liniowy - splot</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Filtr Sobel</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="toolsLabel">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>10</y>
     <width>191</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Narzędzia</string>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="swapButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>654</x>
     <y>490</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Użyj przetw...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="restoreButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>490</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Przywróć oryg...</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
